src/cider/exec/nextgen/jitlib/base/JITFunction.h

```diff
@@ -30,6 +30,21 @@
 namespace cider::jitlib {
 enum JITFunctionParamAttr : uint64_t {};
 
+enum ERROR_CODE {
+  ERR_DIV_BY_ZERO = 1,
```

Comment:
what about 2,5,and others ?

Reply:
and add NO_ERR = 0?

Reply:
I copied these error codes from `Execute.h` directly.

Reply:
There will be a default return value **0** when we build the func, is `NO_ERR = 0` necessary?